easybuild/easyconfigs/f/FFLAS-FFPACK/FFLAS-FFPACK-2.2.0-foss-2016a.eb

```diff
@@ -0,0 +1,46 @@
+##
+# This file is an EasyBuild recipe; see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright (c) 2016 Riccardo Murri <riccardo.murri@gmail.com>
+# Authors::   Riccardo Murri <riccardo.murri@gmail.com>
+# License::   GPL
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'FFLAS-FFPACK'
+version = '2.2.0'
+
+homepage = 'https://linbox-team.github.io/fflas-ffpack/'
+description = "Finite Field Linear Algebra Subroutines / Package"
+
+toolchain = {'version': '2016a', 'name': 'foss'}
+
+sources = ['v2.2.0.zip']
+source_urls = ['https://github.com/linbox-team/%(namelower)s/archive']
+
+builddependencies = [
+    ('Autoconf', '2.69', '', True),
+    ('Automake', '1.15', '', True),
+    ('libtool', '2.4.6', '', True),
+]
+dependencies = [
+    ('GMP', '6.1.0', '', ('GCC', '4.9.3-2.25')),
+    ('Givaro', '4.0.1'),
+]
+
+preconfigopts = "env NOCONFIGURE=1 ./autogen.sh && "
```

Comment:
> I'm not sure how
>
> ```python
> preconfigopts = "env NOCONFIGURE=1 ./autogen.sh && "
> ```
>
> is any different from
>
> ```python
> preconfigopts = "NOCONFIGURE=1 ./autogen.sh && "
> ```

It is different, in that the second form uses a feature of the Bash
shell (more generally of all Bourne-type shells), that variable
assignments preceding a command are automatically exported to that
command.

The first form uses the `env` command, which was born to provide these
feature to shells that do not have it built-in (e.g., tcsh) and is
therefore more robust.